F/FighterBomber/usr/src/FighterBomberHD.s

```diff
@@ -537,15 +549,12 @@ pl_main
 
 	PL_PS	$0c012,use_fmode3
 
-	PL_PS	$0b05e,keyboard_hook
-
+	PL_PS	$0b026,keyboard_hook
 
 	PL_PS	$143c6,cpu_dependent_loop_d2
 	PL_PS	$143f8,cpu_dependent_loop_d0
 	PL_PSS	$1464a,cpu_dependent_loop_d0_2,2
 	
-	PL_PS	$179d8,fix_smc_address

```

Comment:
why having removed "fix_smc_address" ? is that a problem?

Reply:
Not a problem, but it's overwritten by smc_179d8 further down, so it's not actually used which is why I removed it.